src/views/virtualmachines/details/VirtualMachinePendingChangesAlert.tsx

```diff
@@ -47,9 +57,7 @@ const VirtualMachinePendingChangesAlert: React.FC<VirtualMachinePendingChangesAl
 
   return (
     <PendingChangesAlert isWarning>
-      {t(
-        'The following areas have pending changes that will be applied when this VirtualMachine is restarted.',
-      )}
+      {alertText}
       <List>
         <PendingChangesBreadcrumb pendingChanges={pendingChangesDetailsTab} />
         <PendingChangesBreadcrumb pendingChanges={pendingChangesSchedulingTab} />
```

Comment:
Can you move `<PendingChangesBreadcrumb pendingChanges={pendingChangesNICsTab} />` to the end of the list of pending changes? That would make  having a separate requirement (live migrate) more logical.